cdap-api/src/main/java/co/cask/cdap/api/dataset/table/Table.java

```diff
@@ -42,6 +42,12 @@
   String PROPERTY_READLESS_INCREMENT = "dataset.table.readless.increment";
 
   /**
+   * Property set to configure name of the column family. The property only applies to implementations that support
+   * it. If not set, a default column family will be used.
+   */
+  String COLUMN_FAMILY = "dataset.table.column.family";
```

Comment:
should we name it PROPERTY_COLUMN_FAMILY to be consistent with other properties here?